hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.cpp

```diff
@@ -579,11 +579,14 @@ bool XmlChangeset::canSend(XmlRelation* relation)
       //  since these are called frequently
       if (member.isNode())
       {
+        //  Special case, node doesn't exist in changeset, it may in database, send it
+        if (member.getRef() > 0 && _allNodes.find(member.getRef()) == _allNodes.end())
+          return true;
```

Comment:
A relation may have a node/way/relation with a positive ID that isn't in the changeset but it may be in the database.  If it isn't the API will reject the changeset at a later time.  Here, allow it to be sent.